amf-aml/shared/src/main/scala/amf/aml/client/scala/AMLElementClient.scala

```diff
@@ -21,4 +28,103 @@ class AMLElementClient private[amf] (protected val configuration: AMLConfigurati
                                  configuration.errorHandlerProvider.errorHandler(),
                                  references)
 
+  /**
+    * Get all instances of registered dialects
+    * @return a Seq of [[Dialect]]
+    */
+  def getDialects(): immutable.Seq[Dialect] = getDialectsByCondition(_ => true)
+
+  /**
+    * Get all names of registered dialects
+    * @return a Seq of names
+    */
+  def getDialectNames(): immutable.Seq[String] = getDialectsByCondition(_ => true).map(_.nameAndVersion())
```

Comment:
Not sure why we provide these methods from our side as the client can simply do it themselves

Reply:
I would remove them

Reply:
Lo hable con Lean y el no le ve uso tampoco. Ahora lo voy a borrar.